src/commands/config/org/set.ts

```diff
@@ -22,6 +22,21 @@ export default class Set extends Command<SetOrgJson> {
   public async run(): Promise<SetOrgJson> {
     const {flags} = await this.parse(Set);
     if (flags['org-id']) {
+      try {
+        await this.api.get(`project/organization/${flags['org-id']}/`);
+      } catch (error) {
+        // Log original error to debug
+        this.debug('GET project/organization/%s/ resulted in error: %O', flags['org-id'], error);
+        if (
+          error instanceof Error &&
+          error.message === 'Not logged in. Run apimetrics login first.'
+        ) {
```

Comment:
I think this is okay, but generally it'd be better to capture the HTTP code from the API and catch this using the 401 or 403

Reply:
I'm not a big fan of it either. I think perhaps a custom error for the API would work here because currently we just use `Error` for everything, that way we can pass the status code